client/src/document/organisms/article-footer/index.scss

```
@@ -14,6 +14,11 @@
     margin-top: 0;
   }
 
+  @media screen and (max-width: $screen-sm) {
```

```suggestion
  @media (max-width: $screen-sm) {
```
Removing `screen and` in https://github.com/mdn/yari/pull/11142/files, so not addding one back in.